README.md

```diff
@@ -1,3 +1,37 @@
-# Linguist
+# Linguist - Repository Key Extraction Tool
 
-Linguist checks language files to verify consistency and integrity.
+## Usage
+
+### Cloning Repositories with SSH Authentication
+
+To run `linguist`, you must specify the **path to your SSH private key** as an argument.
+
+```sh
+cargo run --bin linguist ~/.ssh/my_custom_rsa_key
```

Comment:
Since liguist runs without the --bin linguist argument, could you please remove it?